### DISCUSSION TITLE
list and discuss missing words here

### DISCUSSION THREAD

**Discussion by gokhaleaaroh** [mukta-strot/marathi-shabd#36] [Ideas]
नमस्कार,

दिवसभरात मराठी बोलत असताना मला असे जाणवले की मला बऱ्याच इंग्रजी शब्दांसाठी मराठी शब्द माहीत नाहीत. त्या मुळे माझा असा एक प्रस्ताव आहे की आपण एक शब्द वाढवण्यासाठी forum बनवूया. दिवसात जर कोन्हाळा असा एक शब्द मनात आला जो db.csv मध्ये भरण्यासारखा असेल, तर ती व्यक्ति forum मध्ये तो शब्द भरूषकते.

**Comment by sanketgarade** [comment]
नक्कीच. शब्द सुचवण्या साठी आपण खाली कमेंट करू शकता. शक्य झाल्यास दोन्ही मराठी व इंग्रजी शब्द लिहा. मराठी शब्द माहीत नसल्यास नुसता इंग्रजी शब्द पण चालेल.

**Comment by sanketgarade** [comment]
भविष्यात मराठी-शब्दच्या "search site" वर आपण एक प्रोग्राम लिहिणार आहे (अधिक माहिती इथे #25) ज्यानेकरून लोकांना थेट database मध्ये शब्द जोडता येतील.

**Comment by gokhaleaaroh** [comment]
word for "probably". I only know "कदाचित", but that fits more to "maybe". Is there a word that fits better for "probably"?

**Reply by sanketgarade** [reply]
बहुतेक?

**Comment by sanketgarade** [comment]
- [ ] खालील शब्द तपासून जोडणे 
चलचित्र - video.
चित्रपट - movie. 
उद्वाहक - lift.
सरकता जिना - escalator.
वाहनथांबा - bus stop. 
पंचा, अंगपुषे - towel 
वाहतूक कंदील - Traffic Signal. 
आकाश रेषा - Skyline.
पथ दिवे - street lamp. 
वरील शब्द बरोबर आहेत का? काही सुधार असल्यास कृपया कळवा. 

[स्त्रोत](https://twitter.com/mahamarathi1/status/1441674552530980877?s=21)

**Reply by rohit-u-s** [reply]
चलचित्र, चित्रपट, उद्वाहक हे बरोबर आहेत 
सरकता जिना, वाहतूक कंदील, पथ दिवे  यांना पर्यायी शब्द मिळू शकतील असे वाटते
towel  ला अजून चांगला पर्यायी शब्द शोधावा लागेल असे वाटते 

मलाही मदत करायला आवडेल 
मी ssc पास आहे 
python शिकायला सुरू केले आहे

**Reply by rohit-u-s** [reply]
उद्-वाहक

**Reply by sanketgarade** [reply]
> मलाही मदत करायला आवडेल 
> 
> मी ssc पास आहे 
> 
> python शिकायला सुरू केले आहे 

नमस्कार रोहित, नक्कीच तुम्ही योगदान करू शकता. शक्य असल्यास [इथे](https://github.com/mukta-strot/marathi-shabd/discussions/8) आपली ओळख करून देऊ शकता.

**Comment by sanketgarade** [comment]
- [ ] नाणेफेक म्हणजे क्रिकेट मधील toss.
